packages/x-charts-pro/.mocharc.js

```diff
@@ -1,34 +0,0 @@
-// We can't import the `.mocharc.js` of the monorepo, otherwise we trigger its `setupBabel`.
```

Comment:
I assuem this is not related to premium, but to the vite test migration

Reply:
Yeah I merged that in another pr, will be gone by the time we merge this 😆